pkg/cmd/adm/unregister_member_test.go

```diff
@@ -15,7 +15,7 @@ func TestUnregisterMemberWhenAnswerIsY(t *testing.T) {
 	toolchainCluster := NewToolchainCluster(ToolchainClusterName("member-cool-server.com"))
 	hostDeploymentName := test.NamespacedName("toolchain-host-operator", "host-operator-controller-manager")
 	deployment := newDeployment(hostDeploymentName, 1)
-	deployment.Labels = map[string]string{"olm.owner.namespace": "toolchain-host-operator"}
+	deployment.Labels = map[string]string{"kubesaw-control-plane": "kubesaw-controller-manager"}
 
 	newClient, fakeClient := NewFakeClients(t, toolchainCluster, deployment)
 	numberOfUpdateCalls := 0
```

Comment:
I'm surprised that this still works without any additional changes - most likely because you restart the member operator (which doesn't contain any deployments).

Reply:
changed the case to accommodate this , and added some comments as i thought  not to duplicate test cases